editoast/src/views/mod.rs

```diff
@@ -1,24 +1,30 @@
 mod infra;
 use rocket::serde::json::Value as JsonValue;
+use rocket_db_pools::deadpool_redis;
 pub mod pagination;
 pub mod params;
 
+use crate::db_connection::{DBConnection, RedisPool};
+use deadpool_redis::redis::cmd;
 use diesel::{sql_query, RunQueryDsl};
 use rocket::{routes, Route};
 use std::collections::HashMap;
 use std::env;
 
-use crate::db_connection::DBConnection;
-
 pub fn routes() -> HashMap<&'static str, Vec<Route>> {
     HashMap::from([("/", routes![health, version]), ("/infra", infra::routes())])
 }
 
 #[get("/health")]
-pub async fn health(conn: DBConnection) -> &'static str {
-    // Check DB connection
```

Comment:
Why remove this comment ?

Reply:
Code is obviously checking this IMO, I can put it back if you find it adds an information

Reply:
I'm okay with this comment deleted.